Cargo.lock

```diff
@@ -12975,7 +12975,7 @@ version = "1.6.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
Is this expected?

Reply:
Let me check it

Reply:
Should be 1.0.0 https://github.com/paritytech/substrate/blob/polkadot-v0.9.30/Cargo.lock#L11349